cmd/neofs-cli/modules/session/create.go

```diff
@@ -30,8 +31,13 @@ const defaultLifetime = 10
 var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create session token",
-	Args:  cobra.NoArgs,
-	Run:   createSession,
+	Long: `Create session token.
+
+Default lifetime of session token is ` + strconv.Itoa(defaultLifetime) + ` epochs 
+if one of --` + commonflags.ExpireAt + ` or --` + commonflags.Lifetime + ` flags is not specified. 
```

Comment:
```suggestion
if none of --` + commonflags.ExpireAt + ` or --` + commonflags.Lifetime + ` flags is specified. 
```

Double-check, please